backend/tests/Feature/OnboardingWizardTest.php

```
@@ -195,4 +196,85 @@ public function test_step2_redirects_without_session(): void
+    {
+        $this->actingAs($this->admin)
+            ->post(route('onboarding.modules'), [
+                'preset' => 'custom',
+                'enabled_modules' => ['not-a-real-module'],
+            ])
+            ->assertSessionHasErrors('enabled_modules.0');
+    }
+
+    public function test_modules_step_requires_admin(): void
+    {
+        $operator = User::factory()->create();
```

PHPStan (2.2.5)

[error] Call to an undefined static method App\Models\User::factory().


(staticMethod.notFound)